setup.py

```diff
@@ -169,7 +169,7 @@ def run(self):
         # Ensure our temporary build directory exists
         try:
             os.makedirs(build_temp)
-        except IOError:
+        except:
```

Comment:
If IOError really can occur, than this should catch the two of them. We are
not going to do a bare except though.

On Sun, Dec 6, 2015 at 9:58 AM, yurivict <notifications@github.com> wrote:

> In setup.py <https://github.com/pyca/pynacl/pull/154#discussion_r46771317>
> :
>
> > @@ -169,7 +169,7 @@ def run(self):
> >          # Ensure our temporary build directory exists
> >          try:
> >              os.makedirs(build_temp)
> > -        except IOError:
> > +        except:
>
> Also: I made a port for FreeBSD. For some reason 'install' operation
> begins to build something. This normally doesn't happen with python ports.
> And python framework is very well behaved and stable in FreeBSD. So there
> is something strange with pynacl.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/pynacl/pull/154/files#r46771317>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084